metadata/config.yaml

```
@@ -36,6 +36,10 @@ CAPELLA_FORK_EPOCH: 56832
 DENEB_FORK_VERSION: 0x90000073
 DENEB_FORK_EPOCH: 132608
 
+# Electra
+ELECTRA_FORK_VERSION: 0x90000074
+ELECTRA_FORK_EPOCH: 219392
```

```suggestion
ELECTRA_FORK_EPOCH: 222464
```

ACDC accepted option 2: https://github.com/ethereum/pm/issues/1265#issuecomment-2637778317